trade-finance-manager-ui/server/controllers/helpers/amendments.helper.js

```diff
@@ -31,15 +12,10 @@ const showAmendmentButton = (deal, userTeam) => {
   const acceptableUser = CONSTANTS.TEAMS.PIM;
   const acceptableStatus = [CONSTANTS.DEAL.DEAL_STAGE.CONFIRMED];
 
-  if (acceptableSubmissionType.includes(deal.dealSnapshot.submissionType) && userTeam.includes(acceptableUser)
-    && acceptableStatus.includes(deal.tfm.stage)) {
+  if (acceptableSubmissionType.includes(deal.dealSnapshot.submissionType) && userTeam.includes(acceptableUser) && acceptableStatus.includes(deal.tfm.stage)) {
```

Comment:
Can we have this compound condition as a helper function please?

Reply:
The function is already inside a helper function/file though. and this validation is not being used anywhere else. it only checks for 3 things